api/geth_backend.go

```diff
@@ -1283,13 +1283,13 @@ func (b *GethStatusBackend) ConvertToKeycardAccount(account multiaccounts.Accoun
 	return nil
 }
 
-func (b *GethStatusBackend) RestoreAccountAndLogin(request *requests.RestoreAccount) (*multiaccounts.Account, error) {
+func (b *GethStatusBackend) RestoreAccountAndLogin(request *requests.RestoreAccount, opts ...params.Option) (*multiaccounts.Account, error) {
```

Comment:
I don't think we wants `opts`, we moved away from this because we don't want clients to directly configure nodeconfig, I see it's used to disable wakuv2, could you elaborate on the rationale?

Reply:
oh yes, I see, but I mean, why use another argument instead of reusing the one we have? it means that we need to look at 2 structs to understand what this endpoint takes as a parameter

Reply:
I made the same complain about create account of login, but got merged somehow :) we should not allow clients to directly manipulate `NodeConfig` (`With...`), as that's quite messy and the point of this new endpoints it to prevent this.
I understand that it's something that you didn't introduce, but merging this commit would double down on the pattern, so not sure it's a good argument, I'd rather remove the pattern, or at least keep the external api unchanged